client/src/pages/Profile/ProfileWidget.jsx

```diff
@@ -1,18 +1,23 @@
 import React from "react";
 import profileStyles from "./ProfileWidget.module.css";
+import { useData } from "../../context/DataContext";
+
 // TODO: make this into arrow function and get rid of curly braces
-const ProfileWidget = ({ profileData, data }) => {
+const ProfileWidget = () => {
+  const { data } = useData();
+  const profile = data[0];
```

Comment:
Since this is pulling data from the new endpoint, any of the data will contain information about the user.

****TODO I just realized a potential bug of if the user has no posts, will add this as another task to handle, will be creating a "get-user-info" endpoint to handle this bug